KlayGE/Core/Include/KlayGE/RenderSettings.hpp

```diff
@@ -34,9 +34,9 @@ namespace KlayGE
 	{
 		RenderSettings()
 			: hide_win(false), full_screen(false),
-				left(0), top(0),
+				left(0), top(0), width(0), height(0),
```

Comment:
These changes are unnecessary. They are guaranteed to be assigned before the first usage.